Covid-19Radar/Covid19Radar
# feature/54

## Purpose

UI Test Scaffolding and basic tests for Xamarin / App Center

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
Adds basic UI Tests


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
* Get the code
* Check the readme on the UI test project for info in how to run test locally and remotely on app center


* Test the code
this feature is adding UI tests :)

## What to Check
Verify that the following are valid
UI test should pass

## Other Information
